opg-php-fpm-1604/docker/confd/templates/app.conf.tmpl

```diff
@@ -14,6 +14,12 @@ server {
     server_name {{ getv "/opg/nginx/server/names" }};
     {{ end }}
 
+    {{ if exists "/opg/nginx/client/max/body/size" }}
```

Comment:
php-pfm image overwrites the app.conf templated, so it's need here too. Now all images based on either base image will work with POST size